fetch_submissions.py

```diff
@@ -69,7 +70,10 @@ def login(username, password):
       'next': '/'
   }
 
-  CLIENT.post(LOGIN_URL, data=payload, headers=dict(Referer=LOGIN_URL))
+  response = CLIENT.post(LOGIN_URL, data=payload, headers=dict(Referer=LOGIN_URL))
+  # The status code and reasons fields for response object return same
+  # values for successful and unsuccessful logins, hence the url field is used
+  return not response.url == "https://leetcode.com/accounts/login/"
```

Comment:
Nit: prefer `!=` over `not __ == __`. It is a very minor nit to be fair. Here's some context: https://stackoverflow.com/a/31026976

Reply:
The efficiency of `!=` as compared to `not __ == __` should not really be the concern here because the time difference is hardly perceptible. I think the point made about the readability of `!=` makes more sense.

Reply:
I had changed the `!=` to `not __ == __` because I thought that the second expression was more Pythonic.